easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-3.0.1_cleanup.patch

```diff
@@ -0,0 +1,282 @@
+# This patch removes all need for the ThirdParty files of OpenFOAM:
```

Comment:
not really, other than I had to repatch it to replicate your 4.0 patch, since the files were different (names, contents)